test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -970,6 +974,145 @@ await CliTester.Run(new RunConfiguration[]
             }, _output);
         }
 
+        [Theory]
+        [InlineData("dotnet")]
+        [InlineData("node")]
+        public async Task Start_MissingLocalSettingsJson_SuccessfulFunctionExecution(string language)
+        {
+            await CliTester.Run(new RunConfiguration[]
+            {
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        $"init . --worker-runtime {language}",
+                        $"new --template Httptrigger --name HttpTriggerFunc",
+                    },
+                    CommandTimeout = TimeSpan.FromSeconds(300),
+                },
+                new RunConfiguration
+                {
+                    PreTest = (workingDir) =>
+                    {
+                       var LocalSettingsJson = Path.Combine(workingDir, "local.settings.json");
+                        File.Delete(LocalSettingsJson);
+                    },
+                    Commands = new[]
+                    {
+                        $"start --{language} --port {_funcHostPort}",
+                    },
+                    ExpectExit = false,
+                    OutputContains = new[]
+                    {
+                        $"local.settings.json",
+                        "Functions:",
+                        $"HttpTriggerFunc: [GET,POST] http://localhost:{_funcHostPort}/api/HttpTriggerFunc"
+                    },
+                OutputDoesntContain = new string[]
```

Comment:
nit: should line up like the others
```suggestion
                    OutputDoesntContain = new string[]
```